aten/src/ATen/native/quantized/cpu/qfc.cpp

```diff
@@ -106,31 +105,31 @@ class QFCInt8 final : public c10::OperatorKernel {
         /*nCol=*/N);
 
     // Allocate output Tensor and a buffer for fbgemmPacked to use
-    auto output = at::zeros({M, N}, input.options().dtype(at::kByte));
+    auto output = _empty_affine_quantized(
```

Comment:
Calls to contiguous() should ideally occur once as a JIT pass so that we do not repeat this operation on constants like bias, please add a TODO for later optimization work.

Reply:
@raghuramank100: which ```contiguous()``` are you referring to? Just add the comment for JIT pass in the previous location (Line 32).